src/client/public/css/components/input.css

```
@@ -10,11 +10,17 @@
 }
 
 .input {
-	padding: 16px;
+	padding: 15px 16px;
 	border-radius: 16px;
 	border: 1px solid var(--color-grey-100);
+	outline: none;
 	font-family: inherit;
 	font-size: inherit;
 	width: 100%;
 	background-color: rgba(175 175 175 / 20%);
 }
+
+.input:hover,
+.input:active {
+	border-color: var(--color-grey-300);
```

```suggestion
	border-color: var(--color-grey-300);
  background-color: rgba(175 175 175 /15%);
```

let's add here backgroun change as well